app/controllers/public_pages_controller.rb

```diff
@@ -50,6 +54,11 @@ def page_not_found
 
   def tax_questions; end
 
+  def stimulus
```

Comment:
The story https://www.pivotaltracker.com/n/projects/2409240/stories/181142024/comments/229655892 seems to say that visitors to demo.getyourrefund.org/stimulus should be redirected to home, but this redirects them to the beginning of intake. Does it make more sense to remove this `PublicPagesController#stimulus` method entirely?

Reply:
Oh cool yay